news/context-fetch-threads

```diff
@@ -0,0 +1,20 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* Allow `CONDA_FETCH_THREADS`/`fetch_threads`, which sets parallel package
+  downloads, to be overridden from its default value of `5`.
```

Comment:
Looks like this news file was included by accident, from https://github.com/conda/conda/pull/13263

Reply:
I merged that PR into this one on purpose.